org.overte.Overte.yml

```diff
@@ -0,0 +1,88 @@
+app-id: org.overte.Overte
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+# QtWebEngine is not part of the KDE runtime,
+# so we use this package, which is also maintained by KDE developers.
```

Comment:
```suggestion
```

Reply:
Are comments like this problematic? They are intended for my colleagues. In this case, it is not obvious that the KDE Qt runtime doesn't ship QtWebEngine since that is part of Qt.

Reply:
Comment bring no value.

Reply:
still applies.

Reply:
I don't see how an extra helpful comment would be problematic.